src/challenges/challengeOne/3.Table/Table.test.tsx

```diff
@@ -1,8 +1,50 @@
-// import React from "react";
-// import { render, screen } from "@testing-library/react";
-// import userEvent from "@testing-library/user-event";
-// import { Table } from "./Table";
-
-// describe("Table", () => {
-//   it("renders...", () => {});
-// });
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import { Table, TableProps } from "./Table";
+
+describe("Table", () => {
+  const columns = [
+    { property: "name", label: "Name" },
+    { property: "age", label: "Age" },
+  ];
+
+  const data = [
+    { id: 1, name: "John Doe", age: 25 },
+    { id: 2, name: "Jane Smith", age: 30 },
+  ];
+
+  const renderComponent = (additionalProps?: Partial<TableProps>) => {
+    const defaultProps: TableProps = {
+      columns,
+      data,
+    };
+
+    render(<Table {...defaultProps} {...additionalProps} />);
+  };
+
+  it("renders the table with column headers", () => {
+    renderComponent();
+
+    const tableElement = screen.getByRole("table");
+    expect(tableElement).toBeInTheDocument();
+
+    columns.forEach((column) => {
```

Comment:
We never want logic in our tests, always test each individual column with its own. expect.